libs/feature/record/src/lib/ign-api-produit/ign-api-produit.component.html

```diff
@@ -0,0 +1,37 @@
+<a
+  class="group flex justify-between card-shadow px-6 py-5 cursor-pointer"
+  rel="noopener"
+>
+  <div class="grow-1 w-full overflow-hidden">
+    <div
+      class="text-16 text-black truncate font-title w-11/12"
+      [title]="link['title'] || link.name"
+    >
+      {{ link['title'] || link.name }}
+    </div>
+    <div class="pt-1">
+      <span
+        class="inline-flex items-center justify-center px-2 py-1 text-13 font-medium leading-none text-white rounded transition-opacity opacity-70 group-hover:opacity-100"
+        [style.background-color]="color"
+        data-cy="download-format"
+        >{{ format || ('downloads.format.unknown' | translate) }}</span
+      >
+      <span
+        class="pl-2 inline-flex items-center text-gray-800 text-sm"
+        *ngIf="isFromWfs"
+        translate=""
+        >datahub.search.filter.generatedByWfs</span
+      >
+    </div>
+    <div *ngFor="let entry of liste$ | async">
+      <a [href]="entry['link'][0]['href']"
+        >{{ entry['link'][0]['href'].split('/').slice(-1) }}
+        <mat-icon
```

Comment:
Sorry, we updated our icon library in the meantime and use `<ng-icon>` now.

Reply:
Yes, I made the change.